### DISCUSSION TITLE
How to connect Win 11 arm guest to a Windows network

### DISCUSSION THREAD

**Discussion by erikmartinessanches** [utmapp/UTM#5060] [UTM Help (macOS)] [answered]
**Describe the issue**  
I'd like to connect Win 11 arm VM Preview version 10.0.25201 inside UTM (M1 host) to a physical, intel-based Win 10 v 10.0.19045 (in Bootcamp) computer. I've enabled network discovery mode and file and printer sharing on both computers. Both are on my home Wifi. UTM network setting: Shared Network.

This is the situation on the VM:
![image](https://user-images.githubusercontent.com/4912935/219907852-d8337e4c-a8a6-4b4d-92c7-e4f29a15d8bc.png)

When I `ping 192.168.64.2` from the physical PC to try to reach the Win11ARM VM, the request times out. Pinging the other way works. I can not see either computer in the "Network" explorer section.

**Configuration**  
* UTM Version: 4.1.5
* macOS Version: 13.2
* Mac Chip (Intel, M1, ...): M1

[config.plist.zip](https://github.com/utmapp/UTM/files/10775506/config.plist.zip)

**Comment by osy** [comment, marked as answer]
This is due to the lack of port forwarding in NAT mode on UTM currently. You can use Emulated VLAN with port forwarding on and then forward the port to your host mac.

**Comment by erikmartinessanches** [comment]
I'm unsure how to configure port forwarding and how to "forward the port to your host mac". It is [like this](https://www.papercut.com/kb/Main/MacPortForwarding)?

**Reply by erikmartinessanches** [reply]
I believe I was actually able to achieve this with Bridged (advanced) network setting.

**Reply by davidvasandani** [reply]
@erikmartinessanches please share the steps to get it working with the Bridged (advanced) network setting. Thank you!
